### PR TITLE
Update PATH prepending with practical example

### PR DESCRIPTION
**Pull request by ssbarnea**
This should save others time as I doubt anyone wanted to added the foo bar example but most wanted to add a folder which is in user home directory, and the ~ expansion does not work in this case. Also this happens to be by far the most likely PATH anyone would want to add.

In fact many would wonder why is not already in the default list.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@ssbarnea Thanks so much for opening a PR! I'll get this triaged for review 🌟

**Review by umutduyum — APPROVED** [review]

**Comment by ssbarnea** [comment]
@janiceilene @umutduyum I think that the stale bot is... misconfigured? It should not add state when nobody did a reviewer that counts toward approval or changes needed.

**Comment by janiceilene** [comment]
@ssbarnea It's currently configured to comment whenever nothing has happened for 7 days, which helps to catch things that may have been missed in triage. This is definitely not stale though, thank so much for your patience as we work our way through all of the amazing contributions ✨

**Comment by ssbarnea** [comment]
No problem, my only worry was that I see often tickets being closed soon after being marked stale due to bots (not this repo). You can call my worry PTSD ;)

**Comment by ssbarnea** [comment]
ping!

**Comment by ssbarnea** [comment]
re-ping... PR vs bureaucracy, who is going to win?

**Review by FAISAILWE23 — APPROVED** [review]

**Review by flawhoodlum1 — APPROVED** [review]

**Review by flawhoodlum1 — APPROVED** [review]

**Comment by martin389** [comment]
Thank you for adding this great example, @ssbarnea 🎉

File: content/actions/reference/workflow-commands-for-github-actions.md [hunk 1]
```diff
@@ -301,11 +301,13 @@ steps:
 
 `echo "{path}" >> $GITHUB_PATH`
 
-Prepends a directory to the system `PATH` variable for all subsequent actions in the current job. The currently running action cannot access the new path variable.
+Prepends a directory to the system `PATH` variable and makes it available to all subsequent actions in the current job; the currently running action cannot access the new path variable. To see the currently defined paths for your job, you can use `echo "$PATH"` in an action.
```
Comment (lucascosti) [inline content/actions/reference/workflow-commands-for-github-actions.md:304]: I'm 99% sure that `>>` appends the new value to the path, not prepends.

```suggestion
Prepends a directory to the system `PATH` variable and makes it available to all subsequent actions in the current job; the currently running action cannot access the updated path variable. To see the currently defined paths for your job, you can use `echo "$PATH"` in a step or an action.
```

File: content/actions/reference/workflow-commands-for-github-actions.md [hunk 2]
```diff
@@ -301,11 +301,13 @@ steps:
...
 
 #### Example
 
+This example demonstrates how to add the user home directory to `PATH`:
```
Comment (lucascosti) [inline content/actions/reference/workflow-commands-for-github-actions.md:308]:
```suggestion
This example demonstrates how to add the user `$HOME/.local/bin` directory to `PATH`:
```

**Review by lucascosti — APPROVED** [review]
A couple of minor improvement suggestions, otherwise ✅ 🎉

**Comment by martin389** [comment]
Looking good in the stage build: https://docs-1781--patch-1.herokuapp.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path

**Comment by martin389** [comment]
The updates can be found here, when available: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path

**Comment by martin389** [comment]
Thanks again @ssbarnea, this was a great practical example to add 😄

**Pull request merged**